statera-backend/cmd/api/domain/db/connection.go

```diff
@@ -0,0 +1,71 @@
+package db
+
+import (
+	"context"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"log"
+)
+
+/*MongoCN is the object that stores the DB connection*/
+var MongoCN = Connect()
+
+var clientOptions = options.Client().ApplyURI("mongodb+srv://admin:admin@stateradb.brnsm.mongodb.net/StateraDB?retryWrites=true&w=majority")
+
+/*Connect is the function that allows the DataBase connection*/
+func Connect() *mongo.Client {
+	client, err := mongo.Connect(context.TODO(), clientOptions)
+	if err != nil {
+		log.Fatal(err.Error())
+		return client
+	}
+
+	err = client.Ping(context.TODO(), nil)
+	if err != nil {
+		log.Fatal(err.Error())
+		return client
+	}
+	log.Println("Connected succesfully with DB")
```

Comment:
La frase está correcta pero lo ideal sería decir "connected *to* DB". Gramaticalmente (al menos en inglés) uno se conecta *a* la base de datos, no *con* la base de datos.